bin/install

```diff
@@ -146,7 +146,21 @@ os_based_configure_options() {
       configure_options="$configure_options --with-readline=$readline_path"
     fi
   else
+    local jpeg_path=$(locate libjpeg.so | head -n 1)
+    local libpng_path=$(locate libpng.so | head -n 1)
     configure_options="--with-openssl --with-curl --with-zlib --with-readline --with-gettext"
+
+    if [ "$jpeg_path" = "" ]; then
+      export ASDF_PKG_MISSING="$ASDF_PKG_MISSING jpeg"
```

Comment:
What are that exports for?

Maybe it could be good to show a nice error message and put info about this requirement on the README. What do you think?

And thanks for the contribution! 🙌